### ISSUE TITLE
Should we change the cover now that we have a stable version?

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/standard#381]
The Standard's stature and international prominence will only increase now that we have a stable version.
We already have reports that the Standard is being used by 6 public orgs in 2 countries, plus requests for translations into French, German, Dutch and Danish. 
Is the current cover the right cover for the Standard's next phase?

**Comment by bvhme** [comment]
Thanks Elena, this version is not a stable version, it is still version 0.2.0.

Apart from that you've not actually described what the issue would be. What exactly makes it not right?

**Comment by ElenaFdR** [comment]
@bvhme I've changed my comment so it's about the first stable version.

Re the issue: I'm concerned that the two hands shaking, with the skin tone tints used, may not translate well internationally (especially outside of Europe). For example, we've already received feedback that the brown hand is hard to parse due to the lack of finger detail (eg, someone told us they thought the pink hand was holding a flip phone.)

I'd love to hear from people in other cultural and institutional contexts whether this cover would be intriguing, neutral, or even a problem.
